Install-Under-CentOS_LAMP.md

```diff
@@ -202,13 +206,30 @@ during the installation.
    slow-query-log-file            = /var/log/mysql/mysql-slow.log
    ```
 
-2. Restart MySQL/MariaDB service to pick up the changes
+   If using MariaDB 11.0 or higher, use the settings below:
+
+   ```shell
+   [mariadb]
+   max_heap_table_size=500M
+   character_set_client=utf8mb4
+   character_set_server=utf8mb4
+   collation_server=utf8mb4_unicode_ci
+   innodb_buffer_pool_size=8G
+   innodb_doublewrite=OFF
+   tmp_table_size=500M
+   log-error                      = /var/log/mysql/mysql-error.log
+   log-queries-not-using-indexes  = 1
+   slow-query-log                 = 1
+   slow-query-log-file            = /var/log/mysql/mysql-slow.log
```

Comment:
I did review, but somehow didn't comment on this.  We should make this consistent, either with no spacing or make them aligned.